alfresco-transformer-base/src/main/java/org/alfresco/transformer/AbstractTransformerController.java

```diff
@@ -167,10 +169,24 @@
                     + "targetExtension: '{}', requestParameters: {}", sourceMimetype, targetMimetype, targetExtension, requestParameters);
         }
 
```

Comment:
You will need to add DIECT_URL to NON_TRANSFORM_OPTION_REQUEST_PARAMETERS on L129.

I know this is still a WIP, so you probably will be refactoring the code so the originalFilename is created so the createTargetFileName is common to both paths.

Reply:
Oh, it have to be there, otherwise it will not work. I had to accidentally removed that during cleaning.